packages/transit-vehicle-overlay/src/BasicModeIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore FIXME: Create TypeScript types for the icons package.
+import Bus from "@opentripplanner/icons/lib/trimet/Bus";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore FIXME: Create TypeScript types for the icons package.
+import Ferry from "@opentripplanner/icons/lib/trimet/Ferry";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore FIXME: Create TypeScript types for the icons package.
+import Streetcar from "@opentripplanner/icons/lib/trimet/Streetcar";
+
+/**
+ * Fill-in type for the icons package.
+ */
+export interface ModeIconProps {
+  mode?: string;
+}
+
+/**
+ * Default, minimal mode icon renderer.
```

Comment:
Is this not already present within the icons package?

Reply:
Should we make the `ModeIcon` prop mandatory then?

Reply:
https://github.com/opentripplanner/otp-ui/pull/470/commits/28312b082dc609ced2e367e583908d4463e4036e removes `BasicModeIcon` and makes the `ModeIcon` prop mandatory, but I'm realizing that it is should not be for cases where users use the `RouteNumberIcon` component instead...